cmd/apps/crossplane_app.go

```diff
@@ -107,14 +106,14 @@ schedule workloads to any Kubernetes cluster`,
 			}
 
 			err := helm3Upgrade(outputPath, "crossplane-alpha/crossplane",
-				namespace, "values.yaml", map[string]string{}, wait)
+				namespace, "values.yaml", "", map[string]string{})
```

Comment:
I’m a bit worried about updating every app at once. For instance this appears to introduce a bug where wait is being ignored as a parameter.

Reply:
wait was not used in the func, so i removed it from the signature and all func calls

Reply:
Then it should be used and/or left.